shared/process/spawn.rb

```diff
@@ -439,22 +475,27 @@
       end
     end
 
+    newline = "\n"
+    platform_is :windows do
+      newline = "\r\n"
+    end
```

Comment:
It seems this should be worth its own `newline_separator` helper, probably in mspec.
I though `File::ALT_SEPARATOR || File::SEPARATOR` would be nice, but it's more complicated and relies on having these constants properly set.
In the end, I think the best is to just move this at the top-level of the file for now, after the `require`, so it can be shared for all usages in this spec, since it seems one of the rare spec to use `\r\n` so far.